src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
 body {
-  margin: 0;
+  $card-border: #f3f3f3;
+  $color-black: #060b35;
+  $color-gray: #616070;
+  $color-blue: #00acdc;
+  $color-white: #fff;
+  $gap: 4px;
+
+  body,
+  p,
+  a,
+  img,
+  div {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: Roboto, Helvetica, sans-serif;
+  }
+
+  .card {
```

Comment:
Create a separate file for each BEM block
![image](https://github.com/mate-academy/layout_product-cards/assets/80700247/82e2fa5a-8b41-4635-adcc-41fe251255cb)